molecule/model/schema_v2.py

```diff
@@ -613,6 +613,9 @@ def pre_validate_base_schema(env, keep_string):
                 'image': {
                     'type': 'string',
                 },
+                'dockerfile': {
```

Comment:
:bomb:

Reply:
What kind of change are you requesting here?

Reply:
Ah right: it's a rebase mixup of some sort. This change is from https://github.com/ansible/molecule/pull/1769, looks like. Happened a few times now so I dropped a rather short bomb ;) Point taken though!